server/api/build.go

```diff
@@ -34,6 +38,48 @@ import (
 	"github.com/woodpecker-ci/woodpecker/server/store"
 )
 
+func CreateBuild(c *gin.Context) {
+	_store := store.FromContext(c)
+	repo := session.Repo(c)
+
+	var p woodpecker.BuildOptions
+
+	err := json.NewDecoder(c.Request.Body).Decode(&p)
+	if err != nil {
+		_ = c.AbortWithError(http.StatusBadRequest, err)
+		return
+	}
+
+	user := session.User(c)
+
+	lastCommit, _ := server.Config.Services.Remote.BranchHead(c, user, repo, p.Branch)
+
+	tmpBuild := &model.Build{
+		Event:     model.EventManual,
+		Commit:    lastCommit,
+		Branch:    p.Branch,
+		Timestamp: time.Now().UTC().Unix(),
+
+		Avatar:  user.Avatar,
+		Message: "MANUAL BUILD @ " + p.Branch,
```

Comment:
What about using the last commit message for this again and maybe adding this prefix?